### PR TITLE
DasharoSystemFeaturesStrings.uni: fix name and desc

### PR DESCRIPTION
**Pull request by wiktormowinski**

### CODE REVIEW

File: DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni [hunk 1]
```diff
@@ -121,9 +121,9 @@
 #string STR_SLEEP_TYPE_S3     #language en-US "Suspend to RAM (S3)"
 #string STR_SLEEP_TYPE_S3_EXPERIMENTAL     #language en-US "Suspend to RAM (S3) - Experimental"
 
-#string STR_POWER_FAILURE_STATE_PROMPT  #language en-US "Power state after power failure"
-#string STR_POWER_FAILURE_STATE_HELP    #language en-US "Selects state the system gets into when power supply is restored after power failure.\n\n"
-                                                          "Note that the definition of power failure varies between platforms and it might take absence of power for up to 30 seconds for this option to apply."
+#string STR_POWER_FAILURE_STATE_PROMPT  #language en-US "Power state when connecting power supply after power failure"
+#string STR_POWER_FAILURE_STATE_HELP    #language en-US "Selects state the system gets into when power supply on desktop platform, and barrel jack / USB-PD charger on laptops is reattached after power failure.\n\n"
```
Comment (miczyg1) [inline DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni]: `on desktop platform, and barrel jack / USB-PD charger on laptops`

This applies to all platforms, not only desktops and laptops... Please drop changes in this line

File: DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni [hunk 2]
```diff
@@ -121,9 +121,9 @@
 #string STR_SLEEP_TYPE_S3     #language en-US "Suspend to RAM (S3)"
 #string STR_SLEEP_TYPE_S3_EXPERIMENTAL     #language en-US "Suspend to RAM (S3) - Experimental"
 
-#string STR_POWER_FAILURE_STATE_PROMPT  #language en-US "Power state after power failure"
-#string STR_POWER_FAILURE_STATE_HELP    #language en-US "Selects state the system gets into when power supply is restored after power failure.\n\n"
-                                                          "Note that the definition of power failure varies between platforms and it might take absence of power for up to 30 seconds for this option to apply."
+#string STR_POWER_FAILURE_STATE_PROMPT  #language en-US "Power state when connecting power supply after power failure"
```
Comment (miczyg1) [inline DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni]: Do we really need this change? The help explains what this option is about

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni (hunk 2 above)
Comment (wiktormowinski) [reply DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni]: it does explain. But then again: https://github.com/Dasharo/dasharo-issues/issues/1021#issuecomment-2304440260

Comment (miczyg1) [reply DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni]:
```suggestion
#string STR_POWER_FAILURE_STATE_PROMPT  #language en-US "Power state after power/AC loss"
```

Does it sound more precise?

Comment (wiktormowinski) [reply DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni]: it is shorter and that's good enough for me (seriously my version took 3 or 4 lines in setup menu, it was ugly)

File: DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni (hunk 1 above)
Comment (miczyg1) [reply DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni]: @wiktormowinski what about here?

File: DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni [hunk 3]
```diff
@@ -121,9 +121,9 @@
 #string STR_SLEEP_TYPE_S3     #language en-US "Suspend to RAM (S3)"
 #string STR_SLEEP_TYPE_S3_EXPERIMENTAL     #language en-US "Suspend to RAM (S3) - Experimental"
 
-#string STR_POWER_FAILURE_STATE_PROMPT  #language en-US "Power state after power failure"
-#string STR_POWER_FAILURE_STATE_HELP    #language en-US "Selects state the system gets into when power supply is restored after power failure.\n\n"
-                                                          "Note that the definition of power failure varies between platforms and it might take absence of power for up to 30 seconds for this option to apply."
+#string STR_POWER_FAILURE_STATE_PROMPT  #language en-US "Power state after power/AC loss"
+#string STR_POWER_FAILURE_STATE_HELP    #language en-US "Selects state the system gets into when power supply is reattached after power failure.\n\n"
```
Comment (miczyg1) [inline DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni]:
```suggestion
#string STR_POWER_FAILURE_STATE_HELP    #language en-US "Selects state the system gets into when power supply is restored after power failure.\n\n"

```

**Review by miczyg1 — CHANGES_REQUESTED** [review]

**Pull request merged**